addons/account/models/chart_template.py

```diff
@@ -171,7 +171,7 @@ def _load(self, template_code, company, install_demo):
         reload_template = template_code == company.chart_template
         company.chart_template = template_code
 
-        if not reload_template:
+        if not reload_template and (not company._existing_accounting() or install_demo):
```

Comment:
Still not convinced with this. Since install_demo is true by default when calling `try_loading()`.

Reply:
I wanted to use `self.env.ref('base.module_account').demo`, which depends on the settings used when installing the module

Reply:
Made the change: better but still not super robust